src/nvim/buffer_defs.h

```diff
@@ -924,6 +932,8 @@ struct diffblock_S {
   diff_T *df_next;
   linenr_T df_lnum[DB_COUNT];           // line number in buffer
   linenr_T df_count[DB_COUNT];          // nr of inserted/changed lines
+  int df_redraw;
+  int is_linematched;
```

Comment:
I was told in the vim pull request to use int's instead of bool for true / false
so to keep the vim and neovim code matching as much as possible, I started using integers
https://github.com/vim/vim/pull/9661?fbclid=IwAR2JGxr3Y-vi9_wqu9MWDTKG2ZIjmbcG4jk-vNcApINkF3XboILSNvxpVC0

Reply:
Booleans is one of the things Neovim explicitly differs from vim on. Even all vim patches have integers converted to booleans.

Reply:
okay, changed